From 6777cb6ae8458f4c6abbaf7480f70dd08848febf Mon Sep 17 00:00:00 2001
From: Vadim Zeitlin <vadim@wxwidgets.org>
Date: Fri, 16 Oct 2026 02:01:47 +0200
Subject: [PATCH] Fix another C++11 build problem due to make_unique<>

Fix another problem identical to the ones fixed by 7b07adf752 (Fix C++11
build due to use of std::make_unique, 2026-10-15).

See #27229.
---
 src/gtk/wayland.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gtk/wayland.cpp b/src/gtk/wayland.cpp
index 0d4fd10d6c93..36ecf9ef4c1e 100644
--- a/src/gtk/wayland.cpp
+++ b/src/gtk/wayland.cpp
@@ -23,6 +23,8 @@
 #include "wx/log.h"
 #include "wx/window.h"
 
+#include "wx/private/make_unique.h"
+
 #include "wx/gtk/private/backend.h"
 
 #include <string.h>
